models/iris/README.md

```diff
@@ -0,0 +1,66 @@
+# Iris classification example
+
+This example uses the [Iris plants dataset](https://scikit-learn.org/stable/datasets/index.html#iris-dataset)
+ provided with sklearn. This is a multi-class classification use case with no
+ favorability (outcomes are not favorable or unfavorable).
+
+ The example shows how to use the Certifai Model SDK to wrap trained models into services
+ so they can be scanned using the
+  {Certifai toolkit](https://cognitivescale.github.io/cortex-certifai/docs/about).
```

Comment:
Last misformatted link! Just need to replace { with [

Reply:
aargh!